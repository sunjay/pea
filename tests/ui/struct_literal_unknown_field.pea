struct Cool {
    pie: [i64],
}

fn main() {
    let value = Cool {pies: [1, 3]};
}
