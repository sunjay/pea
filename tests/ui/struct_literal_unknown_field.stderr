[tests/ui/struct_literal_unknown_field.pea:6:23-6:26] error: `Cool` has no field named `pies`

[tests/ui/struct_literal_unknown_field.pea:6:17-6:20] error: missing field `pie` in initializer of `Cool`

error: aborting due to 2 previous errors

